app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -134,35 +149,29 @@ class GithubWebhook extends RequestHandler<Body> {
 
   Future<void> _scheduleIfMergeable(
     PullRequestEvent event, {
-    @required bool cancelRunningBuilds,
     @required List<IssueLabel> labels,
   }) async {
-    assert(cancelRunningBuilds != null);
-    if (cancelRunningBuilds) {
-      await _cancelLuci(
-        event.repository.name,
-        event.number,
-        event.pullRequest.head.sha,
-        'Newer commit available',
-      );
-    }
     // The mergeable flag may be null. False indicates there's a merge conflict,
     // null indicates unknown. Err on the side of allowing the job to run.
 
     // For flutter/flutter tests need to be optimized before enforcing CQ.
-    bool runCQ = true;
     if (needsCQLabelList.contains(event.repository.fullName.toLowerCase())) {
-      runCQ = await _checkForCqLabel(labels);
+      if (!await _checkForCqLabel(labels)) {
```

Comment:
Do nothing if the CQ label is required and it's missing.

This could mean that we fail to cancel builds when the CQ label is dropped on flutter/flutter.  I think that's ok for now, and tracking whether it used to be set would make this whole thing stateful and more complicated than it already is.